content/_index.md

```diff
@@ -0,0 +1,13 @@
+---
+title: ""
+date: 2018-06-01T14:54:05-07:00
+
+hero_title: "Program your Cloud Applications and Infrastructure"
+hero_description: "Use your favorite languages to build, deploy, and manage cloud software – on any cloud – with one workflow for developers and operators alike."
+hero_buttons: "<a href='/why-pulumi/delivering-cloud-native-infrastructure-as-code/' class='button orange'>Why Pulumi</a>"
+hero_mininav: 'true'
+hero_codeblock: 'true'
+code_display: 'true'
+---
+
+this?
```

Comment:
Is this needed?

```suggestion
```

Reply:
Ah, bit of layout debuggery I forgot to remove, thanks.